DatadogSessionReplay/Sources/Writer/Models/SRDataModels+UIKit.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  File.swift
+//  
+//
+//  Created by Maciej Burda on 13/04/2023.
+//
+
```

Comment:
blocking/ This header and some linter failure on our [custom rule](https://github.com/DataDog/dd-sdk-ios/blob/dd3fc8077342c90a84b825455cedb9a4a9a8e7f5/tools/lint/sources.swiftlint.yml#L80-L88):
```
⛔️ Line 25: `UIApplication.shared` is unavailable in some environments. Check `UIApplication.managedShared`.
Done linting! Found 1 violation, 1 serious in 352 files.
```
See https://github.com/DataDog/dd-sdk-ios/pull/374 for more context on that rule ☝️